includes/promoted-jobs/class-wp-job-manager-promoted-jobs-api.php

```diff
@@ -104,6 +132,45 @@ public function register_routes() {
 				],
 			]
 		);
+		register_rest_route(
+			self::NAMESPACE,
+			self::REST_BASE . '/refresh-status',
+			[
+				[
+					'methods'             => WP_REST_Server::CREATABLE,
+					'callback'            => [ $this, 'refresh_status' ],
+					'permission_callback' => '__return_true',
+				],
+			]
+		);
+	}
+
+	/**
+	 * Adds no-cache headers to the REST response if they're in the Promoted Jobs API namespace.
+	 *
+	 * @param WP_REST_Response $response The response data.
+	 * @param WP_REST_Server   $server The REST server instance.
+	 * @param WP_REST_Request  $request The request used to generate the response.
+	 *
+	 * @return WP_REST_Response The response data.
+	 */
+	public function add_nocache_headers( $response, $server, $request ) {
+		// Check if the request belongs to the specified namespace and the response is successful.
+		if ( str_starts_with( $request->get_route(), '/' . self::NAMESPACE ) && $response->get_status() >= 200 && $response->get_status() < 300 ) {
```

Comment:
I think we should also concat the `self::REST_BASE` in the check, right?

Reply:
Good catch! Fixed in 25e30e573c44a85e6bc223fa294aadf3735ca5fb. Thanks!